.github/workflows/build.yml

```diff
@@ -0,0 +1,58 @@
+name: Build Timecop
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+  release:
+    types: [created]
+jobs:
+  build:
+    name: PHP ${{ matrix.php-version }} (ZTS ${{ matrix.php-zts }})
+    runs-on: ubuntu-20.04
+    strategy:
+      fail-fast: false
+      matrix:
+        php-version:
+          - '5.6'
+          - '7.0'
+          - '7.1'
+          - '7.2'
+          - '7.3'
+          - '7.4'
+        valgrind: [false]
```

Comment:
Any point of this if its always `false`?

Reply:
It's left as a placeholder, for what ever reason when run on PHP 7.3+ it starts finding memory leaks in PHP. :cry: